Test/Install-WhiskeyDotNetTool.Tests.ps1

```diff
@@ -252,69 +286,100 @@ Describe 'Install-WhiskeyDotNetTool.when existing global.json contains invalid J
+        GivenDotNetSuccessfullyInstalls
+        WhenInstallingDotNetTool
+        # ThenVersionToInstall -ExpectedVersion '2.1.526'
     }
+}
 
-    Describe 'Install-WhiskeyDotNetTool.when specified version of DotNet exists globally' {
-        It 'should use global version' {
-            Init
-            GivenGlobalDotNetInstalled '2.1.505'
-            GivenVersion '2.1.505'
-            WhenInstallingDotNetTool
-            ThenReturnedValidDotNetPath
-            ThenGlobalJsonVersion '2.1.505'
-            ThenDotNetNotLocallyInstalled '2.1.505'
-        }
+Describe 'Install-WhiskeyDotNetTool.when installing DotNet and with invalid rollforward value' {
+    It 'should use disabled for rollforward' {
+        Init
+        GivenGlobalJson -Version '2.1.500' -RollForward 'invalid'
+        GivenDotNetSuccessfullyInstalls
+        Mock -CommandName 'Resolve-WhiskeyDotNetSdkVersion' -ModuleName 'Whiskey' -MockWith { '2.1.500' }
+        WhenInstallingDotNetTool
+        ThenVersionToInstall -ExpectedVersion '2.1.500'
+        Assert-MockCalled -CommandName 'Resolve-WhiskeyDotNetSdkVersion' `
+                          -ModuleName 'Whiskey' `
+                          -Times 1 `
+                          -ParameterFilter { $RollForward -eq [Whiskey.DotNetSdkRollForward]::Disable }
     }
+}
 
-    Describe 'Install-WhiskeyDotNetTool.when installing DotNet and global.json exists in both install root and working directory' {
-        It 'should update working directory''s global.json file' {
-            Init
-            GivenGlobalDotNetInstalled '1.1.11'
-            GivenWorkingDirectory 'app'
-            GivenGlobalJsonSdkVersion '1.0.1' -Directory $workingDirectory
-            GivenGlobalJsonSdkVersion '2.1.505' -Directory $TestDrive.FullName
-            GivenVersion '1.1.11'
-            WhenInstallingDotNetTool
-            ThenReturnedValidDotNetPath
-            ThenDotNetNotLocallyInstalled
-            ThenGlobalJsonVersion '1.1.11' -Directory $workingDirectory
-            ThenGlobalJsonVersion '2.1.505' -Directory $TestDrive.FullName
-        }
+
+Describe 'Install-WhiskeyDotNetTool.when specified version of DotNet does not exist globally' {
+    It 'should install .NET Core locally' {
+        Init
+        GivenDotNetSuccessfullyInstalls
+        GivenVersion '2.1.505'
+        WhenInstallingDotNetTool
+        ThenReturnedValidDotNetPath
+        ThenDotNetLocallyInstalled '2.1.505'
     }
 }
-finally
-{
-    ThenRestoreOriginalPathEnvironment
+
+Describe 'Install-WhiskeyDotNetTool.when specified version of DotNet exists globally' {
+    It 'should use global version' {
+        Init
+        GivenGlobalDotNetInstalled '2.1.505'
+        GivenGlobalDotNetHasValidVersion -Version '2.1.505'
+        GivenVersion '2.1.505'
+        GivenDotNetSuccessfullyInstalls
+        WhenInstallingDotNetTool
+        ThenDotNetLocallyInstalled '2.1.505'
+        ThenReturnedValidDotNetPath
+        ThenDotNetWasInstalled

```

Comment:
If the scenario is `'Install-WhiskeyDotNetTool.when specified version of DotNet exists globally'` then this should be testing the opposite, that Whiskey _**did not**_ install a local version.